flake.nix

```
@@ -0,0 +1,55 @@
+{
+  description = "Basti - Securely connect to AWS resources in private VPCs";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+
+        basti = pkgs.buildNpmPackage {
+          pname = "basti";
+          version = "1.7.2";
```

Let's use package version from package.json, this way it will always be in sync with the actual basti-cli version.

```suggestion
        pkgs = nixpkgs.legacyPackages.${system};
        packageJson = builtins.fromJSON (builtins.readFile ./packages/basti/package.json);

        basti = pkgs.buildNpmPackage {
          pname = "basti";
          version = packageJson.version;
```